lib/node_modules/@stdlib/math/stats/anova1/lib/mean_table.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+// MODULES //
+
+var incrstdev = require( '@stdlib/math/stats/incr/stdev' );
+var uniqueElems = require('./unique_elems.js');
+
+/**
+* Computes the vector of means for each treatment.
+*
+* @param {NumericArray} x - Measured values
+* @param {Array} factor - Array of treatments
+* @returns {{}} destination array
```

Comment:
`@returns {{}}` should be `@returns {Object}`. The description `destination array` also needs to be updated.

Reply:
Both resolved.